### ISSUE TITLE
Downloadable files size issue.

### ISSUE THREAD

**Issue by stvene3004** [LuigiBlood/sat_wave#20]
Say, is it normal for SatellaWave to cry about downloadable files being bigger than 512 KB while the files themselves are already smaller than 512 KB? That's what has been happening when I try the files as a "PSRAM" memory type and a "Memory Pack (Free)" memory type. It just won't let me export.

**Comment by LuigiBlood** [comment]
Ah, nope, it shouldn't happen for anything smaller than 512 KB. I'll try to fix this soon.

**Comment by stvene3004** [comment]
Yo, I think I've found the actual problem. I forgot to mention that I was trying to convert ROMs and stuff. So when I converted Super Mario World, I thought I should try using a header from some BS Mario games. Most of those games (Including BS Super Mario USA 1-4 and Collection 3) are above 512 KB, and uses most of the same headers. So when I pasted the part of that header, SatellaWave seemed to have read it as a game above 512 KB. After finding that out, I decided so follow a _particular_ tutorial on YouTube about converting ROMs to SatellaWave downloadables, and now it works just fine.
